Formula/d/d2.rb

```diff
@@ -1,8 +1,8 @@
 class D2 < Formula
   desc "Modern diagram scripting language that turns text to diagrams"
   homepage "https://d2lang.com/"
-  url "https://github.com/terrastruct/d2/archive/refs/tags/v0.6.4.tar.gz"
-  sha256 "42a941c36dab75cc3ffa4dc29f50af89be5f76d252be1bca3d812271f63e30c7"
+  url "https://github.com/terrastruct/d2/archive/refs/tags/v0.6.5.tar.gz"
+  sha256 "757dba84597dd340eca5e59f6e49bd8e1fa29b353022dc231307b01bcf5f0ea2"
```

Comment:
the sha actually changed when the PR created
```suggestion
  sha256 "bb54639c1e9711fd6d2c88f15abd5d3a31b657d5b182154db60a32ee93ee2713"
```

Reply:
I might still feel like we should track the latest release, but let's see how it goes next time.